sugar-event-calendar/includes/admin/assets/images/sugar-calendar-logo-light.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
cc @amdrew for consult: any idea if this SVG needs to be compressed?

I have a fuzzy memory about omitting doctypes and rounding values to reduce file size, but I could be mistaken.

Reply:
@JJJ I can definitely optimize the SVG. While working on this, I exported the SVG in several different ways, and also embedded it in several different ways. I couldn't decide how much CSS control I wanted, if I wanted to go inline or `img`, etc. Eventually I just stopped obsessing and moved on with the project.

Now's a good time to go back and optimize. So thanks for the reminder! All of the other suggestions are perfect. Gonna hit everything.

Thank you sir!